<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>337</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="modelTypeComboBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>chr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>prp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>env</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>bld</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>veh</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>efx</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ldr</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="assetNameLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>118</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="placeholderText">
    <string>assetName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="geoDescLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>118</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x</string>
   </property>
   <property name="placeholderText">
    <string>geoDescription</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numTypeComboBox">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>30</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Previously, this could also be set to &quot;numeric&quot; and &quot;alpha.&quot; These choices have been removed from the UI, but the setting remains as a placeholder.</string>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="geoTypeComboBox">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>30</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>mesh</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>xmesh</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>vrp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ffx</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>phx</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>krak</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cloth</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hair</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lgt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ctl</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pflow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fpsource</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fpsurf</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>fpgeo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>vvg</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="resTypeComboBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>hi</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>md</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>lo</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="dispTypeComboBox">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>disp2d</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>disp3d</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>dispNo</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="subTypeComboBox">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>30</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sub0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sub1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sub2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sub3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="rndrTypeComboBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>X</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>nrndr</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Asset Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Geo Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Num. Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resolution</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>485</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Geo Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Displacement</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subdivision</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>90</y>
     <width>766</width>
     <height>16</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>115</y>
     <width>766</width>
     <height>166</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="rowCount">
    <number>2</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>375</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>27</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>17</number>
   </attribute>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Before</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>After</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>300</y>
     <width>100</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>285</y>
     <width>766</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="geoNumCheckBox">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>60</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto-number Geo Descriptions</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
